src/components/user/user.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import {
+  Link,
+  BrowserRouter,
+  Route,
+  Switch,
+  withRouter,
+  browserHistory
+} from "react-router-dom";
+import {connect} from 'react-redux';
+let url =  'https://flatearth-api.herokuapp.com/api/v1/auth/secret/'
+
+
+class User extends Component{
+  
+  render(){
+  
+    
+const {token1} = this.props;
+var Li = ()=>{
+   console.log(`token1`, token1)
+  var obj = token1;
```

Comment:
token1 is object ?? it's really weird and hard to understand it. The token could be the string. You need to rename it